File: urionlinejudge/1009 - Salary with Bonus.cpp
// Created by islam elhady on 9/5/20.

#include <iostream>
 
using namespace std;
 
int main() {
 
   string name;
  double fixedSalary, totalSold;

  cin >> name;
  scanf("%lf\n%lf", &fixedSalary, &totalSold);

  printf("%s = R$ %.2lf\n", "TOTAL", fixedSalary + (totalSold * 0.15));

  return 0;
}

File: urionlinejudge/1009 - Salary with Bonus.java
// Created by islam elhady on 1/20/18.

import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        
         Scanner s = new Scanner(System.in);
        
               String name;
               double fSalary , tSell , bounse , tSalary;
               name = s.next();
               fSalary = s.nextDouble();
               tSell = s.nextDouble();               
               bounse = (tSell*15/100);
               tSalary = fSalary+bounse;
               System.out.printf("TOTAL = R$ %.2f\n", tSalary);
 
    }
 
}